### PR TITLE
Add Kubecon demo page

### PR DESCRIPTION
**Pull request by szihai**
This PR has three parts:
a. Kubecon 2020 NA Demo instruction README
b. The provider.yaml for crossplane and rds workload definition
c. Crossplane 0.13 helm chart without the OAM crds.

It is different from the recorded demo instruction in 3 areas:
1. It points to 0.0.9 release (instead of 0.08), and
2. It assumes that in release 0.0.9, the default installed webservice workload will have the cue template used in the demo.
3. Changed name of Appfile from vela.yml to vela.yaml

### CODE REVIEW

**Comment by wonderflow** [comment]
all docs removed from `documentation` to `docs`, please change the dir thanks

**Comment by resouer** [comment]
@szihai I noticed there is a `examples` dir in kubevela repo, seems to be the perfect home for this demo. /cc @wonderflow

File: examples/kubecondemo/vela.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+version: "1.0-alpha.1"
+name: lab3
+
+services:
+  database:
+    type: rds
+    name: alibabaRds
+
+  data-api:
+    type: webservice
```
Comment (hongchaodeng) [inline examples/kubecondemo/vela.yaml:10]: The webservice config isn't aligned with the current CUE template in upstream. Could you upload your webservice template here too?

File: examples/kubecondemo/script/webservice.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+kind: WorkloadDefinition
+metadata:
+  name: rds
+  annotations:
+    definition.oam.dev/apiVersion: "database.example.org/v1alpha1"
+    definition.oam.dev/kind: "PostgreSQLInstance"
+    definition.oam.dev/description: "rds on Alibaba Cloud"
+spec:
+  definitionRef:
+    name: rds.apps
```
Comment (hongchaodeng) [inline examples/kubecondemo/script/webservice.yaml:11]: The webservice.yaml is still rds template :)
please update

**Review by hongchaodeng — APPROVED** [review]
lgtm

**Pull request merged**